### ISSUE TITLE
Remove function: Sprite::TouchingColor

### ISSUE THREAD

**Issue by JiepengTan** [goplus/spx#423]
1. This interface is pretty confusing because it makes users enter exact pixel values. From a human point of view, it's hard to tell the difference between RGB(255, 0, 0) and RGB(254, 0, 0). But the code needs these exact numbers to work right.

2. It'd make more sense to just use something like "touching sprite."

3. The performance isn't great either, since it has to pull images from the GPU or go through each pixel one by one.

In short, we don't really need this function, so it should probably be removed.

```py
onKey KeySpace, => {
	if touchingColor(RGB(255, 0, 0)) {
		println "touchingColor Red"
	} else {
		println "no touchingColor Red"
	}
}
```

**Comment by JiepengTan** [comment]
@xushiwei @nighca

**Comment by nighca** [comment]
@JiepengTan It was added for compatibility with the Scratch API [Touching Color](https://en.scratch-wiki.info/wiki/Touching_Color_()%3F_(block)). Therefore, we need to retain it to make it possible to do auto-transforming from Scratch project to spx project.

While I agree with the points you mentioned above, it isn't an API we recommend to use in standard spx projects.
